egs/wsj/s5/steps/nnet2/train_convnet_accel2.sh

```diff
@@ -284,7 +284,7 @@ SoftmaxComponent dim=$num_leaves
 EOF
   
   cat >$dir/replace.1.config <<EOF
-Convolutional1dComponent input-dim=$pool_out_dim output-dim=$conv_out_dim2 learning-rate=$initial_lrate param-stddev=$stddev bias-stddev=$bias_stddev patch-dim=$patch_dim2 patch-step=$patch_step2 patch-stride=$patch_stride2
+Convolutional1dComponent input-dim=$pool_out_dim output-dim=$conv_out_dim2 learning-rate=$initial_lrate param-stddev=$stddev bias-stddev=$bias_stddev patch-dim=$patch_dim2 patch-step=$patch_step2 patch-stride=$patch_stride2 rearrange-input=true
```

Comment:
You might want to rethink what is the default vectorization of the input. Every convolution component other than the first convolution component in the network has cft vectorized input. You might want to make this the default vectorization and rearrange-input when fct input is used.